lib/core/form/components/inplace-mat-form-field-wrapper/inplace-mat-form-field-wrapper.component.scss

```diff
@@ -0,0 +1,31 @@
+.adf-inplace-mat-form-field-wrapper {
+    .mat-form-field-underline {
+        display: none;
+    }
+
+    .mat-form-field-infix {
+        display: flex;
+        border-top: 0;
+    }
+
+    &-error {
+        input {
+            border: 1px solid var(--theme-warn-color) !important;
+        }
+    }
+
+    input {
```

Comment:
This will change all the input under that element, maybe a more precise selection for the element, would you mind use something a little more specific for the element you want to restyle?

Reply:
We are wrapping the `mat-form-field` e.g. 

```
<adf-inplace-mat-form-field-wrapper>
   <mat-form-field>
      <input>
   </mat-form-field>
 </adf-inplace-mat-form-field-wrapper>
```

If we would like to use some more specific class for input, everyone that would use this wrapper would be forced to add this class in order to make it work (most probably they would have to check in code why it is not working and then add this class)

Reply:
as discussed, changed to component